### PR TITLE
[10][MIG] account_invoice_view_payment

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

File: account_invoice_view_payment/views/account_invoice_view.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        <field name="name">account.invoice.form</field>
+        <field name="model">account.invoice</field>
+        <field name="inherit_id" ref="account.invoice_form"/>
+        <field name="arch" type="xml">
+            <field name="payments_widget" position="after">
+                <field name="payment_ids" invisible="1"/>
+                <button type="object"  name="action_view_payments"
+                        class="oe_link" string="View Payments"
+                        colspan="2"
+                        attrs="{'invisible':[('payment_ids','=',[])]}"/>
```
Comment (LoisRForgeFlow) [inline account_invoice_view_payment/views/account_invoice_view.xml]: I would put the group `group_account_user` to every of these buttons as the payments are only visible to this group in standard odoo.

cc @jbeficent

Comment (AaronHForgeFlow) [reply account_invoice_view_payment/views/account_invoice_view.xml]: I agree with @lreficent

Comment (MiquelRForgeFlow) [reply account_invoice_view_payment/views/account_invoice_view.xml]: Done

**Review by LoisRForgeFlow — APPROVED** [review]
Thanks, it looks good now

**Review by JordiBForgeFlow — APPROVED** [review]

**Comment by JordiBForgeFlow** [comment]
@darshan-serpent @rgarnau can you review?

**Review by nikul-serpentcs — APPROVED** [review]
Functional review :+1: 

Nicely done, a nice feature to view payments directly from your invoices!

**Review by rgarnau — APPROVED** [review]
Code and functional review :+1:

File: account_invoice_view_payment/views/account_payment_view.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    <record id="view_account_payment_invoice_form" model="ir.ui.view">
+        <field name="name">account.payment.invoice.form</field>
+        <field name="model">account.payment</field>
+        <field name="inherit_id"
+               ref="account.view_account_payment_invoice_form"/>
+        <field name="arch" type="xml">
+            <button name="post" position="after">
+                <button name="post_and_open_payment" class="oe_highlight"
+                        string="Validate &amp; View Payment" type="object"/>
```
Comment (serpentcs-dev1) [inline account_invoice_view_payment/views/account_payment_view.xml]: @mreficent Don't you think you also need to add the same group `groups="account.group_account_user"/>` here ?!

**Review by serpentcs-dev1 — APPROVED** [review]
Code & Functional Review LGTM :+1:  

Just one minor thing.

**Comment by JordiBForgeFlow** [comment]
Thanks to all! Now merging

**Pull request merged**